pkgs/build-support/bintools-wrapper/ld-wrapper.sh

```
@@ -44,6 +44,8 @@ if [[ "${NIX_ENFORCE_PURITY:-}" = 1 && -n "${NIX_STORE:-}"
         p2=${params[n+1]:-} # handle `p` being last one
         if [ "${p:0:3}" = -L/ ] && badPath "${p:2}"; then
             skip "${p:2}"
+        elif [ "${p:0:10}" = /LIBPATH:/ ] && badPath "${p:9}"; then
+            skip "${p:9}"
```

Actually, we should reject these, rather than skipping them, for consistency.